README.md

```diff
@@ -19,6 +19,27 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
+## Building 
+Install bazel:
+
+```
+sudo apt-get update
```

Comment:
That can potentially be removed. Updating is good practice but I wouldn't put it here. I don't mind if it stays, though.